packages/main/src/webComponents/Input/Input.stories.tsx

```diff
@@ -1,33 +1,39 @@
 import { action } from '@storybook/addon-actions';
-import { boolean, select, text } from '@storybook/addon-knobs';
+import { boolean, number, select, text } from '@storybook/addon-knobs';
 import { Input } from '@ui5/webcomponents-react/lib/Input';
 import { InputType } from '@ui5/webcomponents-react/lib/InputType';
+import { SuggestionItem } from '@ui5/webcomponents-react/lib/SuggestionItem';
 import { ValueState } from '@ui5/webcomponents-react/lib/ValueState';
-import { Icon } from '@ui5/webcomponents-react/lib/Icon';
 import React from 'react';
 
 export default {
   title: 'UI5 Web Components / Input',
-  component: Input
+  component: Input,
+  parameters: {
+    subcomponents: { SuggestionItem }
+  }
 };
 
 export const generatedDefaultStory = () => (
   <Input
     disabled={boolean('disabled', false)}
-    placeholder={'A placeholder text...'}
+    maxlength={number('maxlength', undefined)}
+    name={text('name', '')}
```

Comment:
Add import for `name` prop: `@ui5/webcomponents/dist/features/InputElementsFormSupport.js`